static/common/js/registration.js

```diff
@@ -0,0 +1,47 @@
+getById = id => document.getElementById(id);
+
+
+let nick = getById("nickname");
+let firstName = getById("first_name");
+let lastName = getById("last_name");
+let age = getById("age");
+let sex = getById("sex");
+let gdpr = getById("gdpr");
+
+getById("after_signature").style.opacity = 0;
+firstName.addEventListener('keyup', () => getById("text_fname").innerHTML = firstName.value);
+lastName.addEventListener('keyup', () => getById("text_lname").innerHTML = lastName.value);
+age.addEventListener('keyup', () => getById("text_age").innerHTML = age.value);
+age.addEventListener('change', () => getById("text_age").innerHTML = age.value);
+
+nick.addEventListener('keyup', () => {
+    let before = ".character::before{content: '" + nick.value + ": '}";
+    let head = document.head.innerHTML;
+    document.head.innerHTML = head + "<style>" + before + "</style>";
+    document.getElementById("signature").innerHTML = nick.value;
+});
+
+sex.addEventListener('change', () => {
+    let sex_suffix = sex.value == 1 ? "a" : "";
+    let spans = document.querySelectorAll('.sex_suffix');
+
+    let motive = "Co tě dovedlo k rozhodnutí zaregistrovat se na náš web? Je nějaký speciální důvod, proč bys rád" + sex_suffix + " patřil" + sex_suffix + " mezi nás?";
+    let source = "Odkud ses dozvěděl" + sex_suffix + " o existenci tohoto města (webu)?";
+
+    getById("motive").placeholder = motive;
+    getById("source").placeholder = source;
+
+    for (span of spans) {
+        span.innerHTML = sex_suffix;
+    }
+});
+
+
+gdpr.addEventListener("change", () => {
+    let agreed = gdpr.value == 1;
+    let opacity = agreed ? 1 : 0;
+    getById("after_signature").style.opacity = opacity;
+});
+
```

Comment:
Please make this file a [javascript module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) that exports a single function, `handleRegistrationForm()` that attaches all event listeners.

Please only attach listeners after [onDomReady](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) has been fired, now you have a race condition.

Reply:
@Almad Could you please see the new version? I am not sure if I did the module like you wanted. Also it seems not to be working if I do this:
```
window.addEventListener('DOMContentLoaded', (event) => {
    handleRegistrationForm();
});
```
but the current version seems to be working. 

Also wanted to do `export default` but it did not work, so I left only `export`.